Dockerfile

```diff
@@ -14,8 +14,8 @@ COPY ./scripts/install-linux-deps.sh /usr/local/src/valhalla/scripts/install-lin
 RUN bash /usr/local/src/valhalla/scripts/install-linux-deps.sh
 
 # get the code into the right place and prepare to build it
-ADD . /usr/local/src/valhalla
-RUN ls
+ADD . .
+RUN ls -la
```

Comment:
this is a remnant from testing when it didn't work, but doesn't hurt to keep either

Reply:
oh i see the workdir is already set there